### ISSUE TITLE
Changed behaviour when retrieving empty ManyToMany relationship

### ISSUE THREAD

**Issue by ChuckBorris33** [vuex-orm/vuex-orm#270]
When updating from version 0.26.4 to version 0.31.1 we encountered changed behaviour when retrieving empty ManyToMany relationships. I can't find this in change log so i might be bug.

Lets say we have model:
```
export default class Todo extends Model {
  static entity = 'todos'

  static fields () {
    return {
      id: this.increment(),
      assignees: this.belongsToMany(User, TodoUser, 'todo_id', 'user_id')
    }
  }
}
```
And we initialise data by:
```
const initialData = [
      {
        id: 1,
        assignees: []
      }
]
this.$store.dispatch('entities/todos/create', { data: initialData })
```
In version 0.26.4 `Todo.query().withAll().find(1).assignees` would return `[]`

In version 0.31.1 `Todo.query().withAll().find(1).assignees` returns `undefined`

I created reproducible example here [https://github.com/ChuckBorris33/vuex-orm-manytomany-bug](https://github.com/ChuckBorris33/vuex-orm-manytomany-bug)

Try to run with version 0.26.4 then install 0.31.1 and try to run.

**Comment by kiaking** [comment]
@ChuckBorris33 Hi! Thank you so much for the report! Well this sure looks like a bug... I'll look into it!

**Comment by ChuckBorris33** [comment]
@kiaking Hi. It's fixed since version 0.31.5. 

Thank you very much !

**Comment by kiaking** [comment]
@ChuckBorris33 Thank you so much for the confirmation! Glad to hear that 👍
